src/printer/components/settings.js

```diff
@@ -68,13 +69,27 @@ function updateApiKey(apiKey) {
   }
 }
 
+function checkOldLink(data) {
+  if (shownOutdatedOs ||
+      typeof data.version.system === "undefined" ||
+      typeof data.version.system.RELEASE === "undefined" ||
+      data.version.system.RELEASE >= 12) {
```

Comment:
Tohle je blbost protože:

- co když to poběží na ARMBianu -> tam ta 11ka nemusí vadit
- a co všechny ostatní tiskárny, taková SL tiskárna může mít klidně release 2 a je to v pořádku. 

Resp. musí to být závislé pouze na PrusaLinku pro MK3, který běží na Rasbianu, dokážeme to?